x/tally/types/abci_types.go

```diff
@@ -26,7 +26,7 @@ type Request struct {
 type RevealBody struct {
```

Comment:
I notice this struct is also missing the `DR_ID`

Reply:
I guess the module technically doesn't make use of it, but good catch. I added a field "ID."